### PR TITLE
CRAN patch 0.6.0.1

### PR DESCRIPTION
**Pull request by mattansb**

### CODE REVIEW

File: R/reexports.R [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+#' @importFrom datawizard ranktransform
+datawizard::ranktransform
+
+#' @export
+#' @importFrom datawizard adjust
+datawizard::adjust
+
+#' @export
+#' @importFrom datawizard change_scale
+datawizard::change_scale
```
Comment (IndrajeetPatil) [inline R/reexports.R:25]: Why are you re-exporting the alias `change_scale()`, which will be removed from `{datawizard}` in future releases?

https://github.com/easystats/datawizard/issues/53

Comment (mattansb) [reply R/reexports.R:25]: Because modelbased is importing it from effectsize 🙃 and I don't want to break it.

Comment (IndrajeetPatil) [reply R/reexports.R:25]: Argh, all these re-exports are making our lives so difficult. We really need to take care of this roadblock soon.

**Comment by mattansb** [comment]
On its way to CRAN

**Pull request merged**